src/stim/stabilizers/tableau.cc

```diff
@@ -379,6 +386,31 @@ bool Tableau::satisfies_invariants() const {
     return true;
 }
 
+bool Tableau::is_conjugation_by_pauli() {
```

Comment:
This method is a const method.

I would name this `is_pauli_product` since the tableau itself is an operation, not conjugation by that operation. You can *apply the tableau in that way*, and that is the most common way to use it, but it's not in and of itself a conjugation.

Reply:
The tableau is defined by conjugated Pauli operators, but the tableau isn't itself a conjugation.

Reply:
done